### ISSUE TITLE
Add "Submit EOI" button to Join popup

### ISSUE THREAD

**Issue by matt-goldman** [SSWConsulting/SSW.Rewards#379]

**Comment by JeanThirion** [comment]
Very good ! I want this !

**Comment by JackDevAU** [comment]
+1 really like this idea! 

Stuff that adds value to the app without being related to prizes!

**Comment by GordonBeeming** [comment]
This is a great idea, I want this too, I think it will make it easier to open the funnel

**Comment by bahjatalaadelssw** [comment]
As long as we can funnel candidates through LinkedIn only, for formatting consistency.

**Comment by PennyWalker** [comment]
Yes! This is a great idea, it'll be unique and a cool way to interact with us!

**Comment by Marxoz** [comment]
Yes I like it. that all :)

**Comment by Berber-Mama** [comment]
That sounds cool and unique, but I'm curious whether someone would use it to apply for a job...
